src/services/SpotifyService.ts

```diff
@@ -29,6 +34,8 @@ export class SpotifyService {
           id: podcastId,
           offset: "0",
           limit: maxResults.toString(),
+          page: page,
+          perPage: perPage,
         },
```

Comment:
Is there a difference between `perPage` and `limit`?

Reply:
The difference is that limit (MAX_RESULTS = 10) is used in the context of defining the maximum number of results that I get using the spotify API, while perPage (itemsPerPage = 8) is used in the context of pagination aka it represents the number of podcasts I want to be displayed on each page.

Reply:
Interesting.
Okay, if this is how it works it looks correct to me